pfio/v2/s3.py

```diff
@@ -605,3 +605,13 @@ def remove(self, file_path: str, recursive=False):
         key = _normalize_key(key)
         return self.client.delete_object(Bucket=self.bucket,
                                          Key=key)
+
+    def normpath(self, file_path: str) -> str:
+        path = os.path.join(self.cwd, file_path)
+        norm_path = _normalize_key(path)
+
+        return "s3(endpoint={})/{}/{}".format(
```

Comment:
endpoint should also be normalized as well; e.g. trailing slash.

Reply:
Fixed by https://github.com/pfnet/pfio/pull/316/commits/0c5974c5d3d264cf45e125b0de2177fcdfbba7ea. Parsed endpoint and use its hostname only